source/plugins/go/src/oms.go

```diff
@@ -1228,6 +1269,8 @@ func PostDataHelper(tailPluginRecords []map[interface{}]interface{}) int {
 				Log(message)
 				continue
 			}
+			elapsed := time.Since(start)
+			Log(fmt.Sprintf("KubernetesMetadata Processing Time: %s", elapsed))
```

Comment:
we would need to emit this as metric to Application insights.

Reply:
added